src/Raven.Server/Documents/TimeSeries/TimeSeriesStats.cs

```diff
@@ -243,7 +252,12 @@ bool TryHandleDeadSegment()
                     start = first.Timestamp;
                 }
 
-                if (baseline <= end && end <= last)
+                var readerOfLastValue = tss.GetReader(context, slicer.DocId, slicer.Name, start, DateTime.MaxValue);
+                readerOfLastValue.Last();
```

Comment:
Is it ok that result returned by `Last()` isn't assigned? Just to confirm.

Reply:
this is only to move the reader to the last segment.